### ISSUE TITLE
Setting up a static blog

### ISSUE THREAD

**Issue by VincentTam** [cirosantilli/china-dictatorship#33]
Hi Ciro,

Thanks for your effort in raising our awareness about PRC's status !!!

Given that your posts are written in Markdown, have you ever considered making this a static web site?  In general, you can simply place everything under the `content` folder and deploy to GitHub/GitLab Pages.

For instance, GitHub has built-in Jekyll support so that users can [build a Jekyll web site in minutes](https://github.com/daattali/beautiful-jekyll/).

**Comment by cirosantilli** [comment]
Hi Vincent,

What's the advantage of a static website? It's more moving parts ;-)

That's what I do for my home page currently: http://www.cirosantilli.com/

But I saw the light and am moving to HUGE SINGLE PAGE ASCIIDOC, that is the ultimate method, FAQ is already in asciidoc, and see this beautiful larger ASCIIDOC example: https://github.com/cirosantilli/linux-kernel-module-cheat

**Comment by VincentTam** [comment]
Currently, this project's content is hosted on a _dynamic_ site.  [A static site is more reliable than a dynamic one](https://www.netlify.com/blog/2016/05/18/9-reasons-your-site-should-be-static/#5-reliability), especially when the traffic is large, since static web servers concentrate on content delivery.

The Markdown source code for posts remains hosted on remote Git repo, so interested readers can find the repo Git repo and star it.

In terms of issues, it can be focused on _technical_ problems of the site.  In the model of static sites, comments/questions about the site's _contents_ can be handled by [Staticman](https://staticman.net).

You may also post a large single page doc and get it rendered by any static site generator.

**Comment by cirosantilli** [comment]
I think github's speed and uptime is enough, if you find concrete evidence that it is not for someone e.g. in china vs a static site, send it my way.

Also I wonder if it wouldn't be easier for china to block cirosantilli.github.io vs github.com/

**Comment by VincentTam** [comment]
> Also I wonder if it wouldn't be easier for china to block cirosantilli.github.io vs github.com/

It would be likely to block the two together.  The former can't persist without the later.  However, as long as you've got the Git repo, you get deploy it to other web hosts.

**Comment by VincentTam** [comment]
Update: rare failure of GitHub when I was about to open an issue.

![Screenshot from 2019-08-13 09-03-21](https://user-images.githubusercontent.com/5748535/62921646-947f3c80-bda9-11e9-9777-e9c72521d32b.png)

**Comment by cirosantilli** [comment]
Yes, I saw something a few minutes ago too. But GitHub status doesn't say anything. Weird.

**Comment by VincentTam** [comment]
This reminds me a classical Chinese idiom "a crafty rabbit has three burrows" (狡兔三窟): https://en.wikipedia.org/wiki/Lord_Mengchang#A_crafty_hare_has_three_burrows.

**Comment by cirosantilli** [comment]
Oh, and they changed IP addresses, `git pull` gave:

> Warning: Permanently added the RSA host key for IP address '140.82.114.3' to the list of known hosts.

**Comment by VincentTam** [comment]
The changing of GitHub's IP address is another example of this four-word proverb, even that's not what  I intended to mean when I left my last comment.
